### PR TITLE
[entropy_src/dv] Changed sanity to smoke

### PR DESCRIPTION
**Pull request by senelson7**
Signed-off-by: Steve Nelson <steve.nelson@wdc.com>

### CODE REVIEW

File: hw/ip/entropy_src/doc/dv_plan/index.md [hunk 1]
```diff
@@ -84,10 +84,10 @@ It creates the following analysis ports to retrieve the data monitored by corres
 ## Building and running tests
 We are using our in-house developed [regression tool]({{< relref "hw/dv/tools/README.md" >}}) for building and running our tests and regressions.
 Please take a look at the link for detailed information on the usage, capabilities, features and known issues.
-Here's how to run a basic sanity test:
+Here's how to run a basic smoke test:
```
Comment (cindychip) [inline hw/ip/entropy_src/doc/dv_plan/index.md:87]: we think it would be good to remove the basic and just use `run a smoke test`

Comment (senelson7) [reply hw/ip/entropy_src/doc/dv_plan/index.md]: Done

**Review by cindychip — APPROVED** [review]
LGTM

**Comment by cindychip** [comment]
@senelson7 I think the private CI failure is due to private CI run script issue. It is trying to support both sanity and smoke. I will try to fix that soon.

**Review by weicaiyang — APPROVED** [review]
LGTM

**Review by mwbranstad — APPROVED** [review]
LGTM

**Review by rasmus-madsen — APPROVED** [review]
LGTM

**Pull request closed without merging**